### DISCUSSION TITLE
How to use https in docsify?

### DISCUSSION THREAD

**Discussion by Minecodes** [docsifyjs/docsify#1506] [Q&A] [answered]
How can i use https in docsify? I want to use it on my website, but don't publish it becouse the https.

**Comment by sy-records** [comment]
you can use nginx or github pages.

**Comment by trusktr** [comment, marked as answer]
https is configured in your server and has nothing to do with Docsify. Docsify only make you a web app. Then how you serve the files depends on you, for now at least.

I recommend looking for tutorials on how to host websites.

Maybe in the future we can provide our own host solution.

**Reply by trusktr** [reply]
BTW! Here is a nice and very simple tool for easily deploying static websites:

https://surge.sh
